search.py

```diff
@@ -283,3 +316,51 @@ def search(input_root, data_tree):
                 # elements don't match->stop input iteration and move to next data element
                 break
     return measure_match_list
+
+
+def get_mei_from_folder(path):
+    """gets a list of MEI files from a given folder path
+    Arguments: path [string]: absolute or relative path to folder
+    Returns: all_mei_files: List<file>: list of mei files in path
+    """
+    return [path + "/" + filename for filename in os.listdir(path) if filename.endswith('.mei')]
+
+
+def text_box_search_folder(path, tag, search_term):
+    """applies the text_box_search() method to a full folder
+    Arguments:  path [string]: absolute of relative path to folder
+                tag [string]: element type
+                search_term[string]: search term to find element
+    Returns:    text_box_array [List<string>]: list of the path of the file that contains the given tag and the measure
+                                                in which it is found
+    """
+    file_list = get_mei_from_folder(path)
+    text_box_array = []
+    for file in file_list:
+        _, root = prepare_tree(file)
+        list = text_box_search(root, tag, search_term)
+        string_list = []
+        if len(list) != 0:
```

Comment:
The if is unnecessary - if the list has 0 length, range will already return the empty sequence.